jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -302,6 +302,7 @@
         export want_neutronsles12=1
         export want_mtu_size=8900
         [[ $libvirt_type = hyperv ]] && want_mtu_size=1500 # windows-PXE-bootloader seems to not like jumbo-packets
+        [[ $hacloud ]] && [[ $(uname -m) = aarch64 ]] && echo "WARNING: there are no HA repos for aarch64" && hacloud=
```

Comment:
because when calling mkcloud without jenkins, it is a lot easier to have hacloud unset yourself. We could still do an early check-and-fail there in addition.